apps/site/src/pages/account/api-keys-panel/api-keys-empty-state.tsx

```diff
@@ -0,0 +1,14 @@
+import { Box, Typography } from "@mui/material";
+
+export const ApiKeysEmptyState = () => {
+  return (
+    <Box>
+      <Typography fontWeight={700} lineHeight={1.5}>
+        You currently do not have any keys.
+        <Typography color="gray.60">
```

Comment:
This is `<p>` within a `<p>`, which is not valid HTML, causing the following console error:

```
next-dev.js:23 Warning: validateDOMNesting(...): <p> cannot appear as a descendant of <p>.
    at p
    at eval (webpack-internal:///../../node_modules/@emotion/react/dist/emotion-element-cbed451f.browser.esm.js:57:66)
    at Typography (webpack-internal:///../../node_modules/@mui/material/Typography/Typography.js:98:87)
    at p
    at eval (webpack-internal:///../../node_modules/@emotion/react/dist/emotion-element-cbed451f.browser.esm.js:57:66)
    at Typography (webpack-internal:///../../node_modules/@mui/material/Typography/Typography.js:98:87)
    at div
    at eval (webpack-internal:///../../node_modules/@emotion/react/dist/emotion-element-cbed451f.browser.esm.js:57:66)
    at Box (webpack-internal:///../../node_modules/@mui/system/esm/createBox.js:34:72)
    at ApiKeysEmptyState
    at div
    at PanelSection (webpack-internal:///./src/components/pages/account/panel-section.tsx:11:11)
    at div
    at eval (webpack-internal:///../../node_modules/@emotion/react/dist/emotion-element-cbed451f.browser.esm.js:57:66)
    at Stack (webpack-internal:///../../node_modules/@mui/material/Stack/Stack.js:118:87)
    at ApiKeysPanel (webpack-internal:///./src/pages/account/api-keys-panel.tsx:36:82)
    at div
    at eval (webpack-internal:///../../node_modules/@emotion/react/dist/emotion-element-cbed451f.browser.esm.js:57:66)
    at Paper (webpack-internal:///../../node_modules/@mui/material/Paper/Paper.js:77:83)
    at div
    at eval (webpack-internal:///../../node_modules/@emotion/react/dist/emotion-element-cbed451f.browser.esm.js:57:66)
    at Box (webpack-internal:///../../node_modules/@mui/system/esm/createBox.js:34:72)
    at div
    at eval (webpack-internal:///../../node_modules/@emotion/react/dist/emotion-element-cbed451f.browser.esm.js:57:66)
    at Container (webpack-internal:///../../node_modules/@mui/system/esm/Container/createContainer.js:116:19)
    at div
    at eval (webpack-internal:///../../node_modules/@emotion/react/dist/emotion-element-cbed451f.browser.esm.js:57:66)
    at Box (webpack-internal:///../../node_modules/@mui/system/esm/createBox.js:34:72)
    at PageContainer (webpack-internal:///./src/components/pages/dashboard/page-container.tsx:11:11)
    at Account (webpack-internal:///./src/pages/account/[[...slugs]].page.tsx:66:74)
    at AuthWallWrapper (webpack-internal:///./src/components/pages/auth-wall.tsx:15:11)
    at NextPageComponent
    at div
    at eval (webpack-internal:///../../node_modules/@emotion/react/dist/emotion-element-cbed451f.browser.esm.js:57:66)
    at Box (webpack-internal:///../../node_modules/@mui/system/esm/createBox.js:34:72)
    at div
    at eval (webpack-internal:///../../node_modules/@emotion/react/dist/emotion-element-cbed451f.browser.esm.js:57:66)
    at Box (webpack-internal:///../../node_modules/@mui/system/esm/createBox.js:34:72)
    at NavbarContainer (webpack-internal:///./src/components/navbar.tsx:683:11)
    at div
    at eval (webpack-internal:///../../node_modules/@emotion/react/dist/emotion-element-cbed451f.browser.esm.js:57:66)
    at Box (webpack-internal:///../../node_modules/@mui/system/esm/createBox.js:34:72)
    at PageLayout (webpack-internal:///./src/components/page-layout.tsx:28:11)
    at SnackbarProvider (webpack-internal:///../../node_modules/notistack/dist/notistack.esm.js:739:24)
    at InnerThemeProvider (webpack-internal:///../../node_modules/@mui/system/esm/ThemeProvider/ThemeProvider.js:21:70)
    at ThemeProvider (webpack-internal:///../../node_modules/@mui/private-theming/ThemeProvider/ThemeProvider.js:43:5)
    at ThemeProvider (webpack-internal:///../../node_modules/@mui/system/esm/ThemeProvider/ThemeProvider.js:40:5)
    at LazyMotion (webpack-internal:///../../node_modules/framer-motion/dist/es/components/LazyMotion/index.mjs:48:23)
    at MyApp (webpack-internal:///./src/pages/_app.page.tsx:85:11)
    at PathnameContextProviderAdapter (webpack-internal:///../../node_modules/next/dist/shared/lib/router/adapters.js:62:11)
    at ErrorBoundary (webpack-internal:///../../node_modules/next/dist/compiled/@next/react-dev-overlay/dist/client.js:301:63)
    at ReactDevOverlay (webpack-internal:///../../node_modules/next/dist/compiled/@next/react-dev-overlay/dist/client.js:850:919)
    at Container (webpack-internal:///../../node_modules/next/dist/client/index.js:62:1)
    at AppContainer (webpack-internal:///../../node_modules/next/dist/client/index.js:172:11)
    at Root (webpack-internal:///../../node_modules/next/dist/client/index.js:347:11)
w
```

Reply:
Fixed it here https://github.com/blockprotocol/blockprotocol/pull/1257/commits/a388f4be030623e79a9742a49817507a5e834667